examples/browser/package.json

```diff
@@ -59,7 +59,7 @@
     "clean": "theia clean && rimraf errorShots",
     "build": "theia build --mode development",
     "watch": "yarn build --watch",
-    "start": "theia start",
+    "start": "export THEIA_DEFAULT_PLUGINS=local-dir:../../plugins && theia start",
```

Comment:
will it be committed ?

Reply:
I found myself doing it over and over, and thought that we can specify relative plugins location for testing purposes. 

Actually, i think maybe we should add a new example that includes only `plugin-ext-vscode` and its dependency, and exclude all irrelevant Theia extensions.

// cc @svenefftinge